### PR TITLE
FI-375 - Hotkeys manipulation

### PR DESCRIPTION
**Pull request by jean-dfinity**
Thank you for your contribution to the IC Developer Portal.
Before submitting your Pull Request, please make sure that:

- [X] You have updated the [`.github/CODEOWNERS`](https://github.com/dfinity/portal/blob/master/.github/CODEOWNERS) file according to your changes.
- [X] You have registered new documents to [`/sidebars.js`](https://github.com/dfinity/portal/blob/master/sidebars.js) if any.

### CODE REVIEW

File: docs/developer-docs/functionality/rosetta/hotkeys.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+
+This page will explain how to generate a hotkey for neuron management.
+The recommended way to get hotkeys is to programmatically 
+generate them using the same process used in the public `ic` repository.
+However, it is possible to generate hotkeys manually as described
+below.
+
+## Manual process
+
+The manual hotkey generation process consists in generating a PEM file then deriving some private and public keys.
```
Comment (sesi200) [inline docs/developer-docs/functionality/rosetta/hotkeys.md:11]:
```suggestion
The manual hotkey generation process consists in generating a PEM file, then deriving the corresponding private and public keys.
```

File: docs/developer-docs/functionality/rosetta/hotkeys.md [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+
+`openssl pkey -inform pem -outform der -in private.pem -out private.der`
+
+Get hex representation:
+
+`xxd -c 48 -s 16 -p private.der`
+
+Those resulting 32 bytes form the secret key.
+
+Follow the same process for the public key.
```
Comment (sesi200) [inline docs/developer-docs/functionality/rosetta/hotkeys.md:33]: Do you mean `openssl pkey -inform pem -outform der -in public.pem -out public.der`
 followed by `xxd -c 48 -s 16 -p public.der`? This feels relatively unclear for a non-insider. Can you maybe substitute `same process` with something like `the last two steps`?

File: docs/developer-docs/functionality/rosetta/hotkeys.md [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+
+
+## FAQ
+
+- Why do I get *"Algorithm ed25519 not found"* while generating the PEM file?
+
+The version of OpenSSL included in MacOS doesn't support ed25519 by default. 
+You may have to install another version of OpenSSL, or run the command from a Linux machine.
```
Comment (sesi200) [inline docs/developer-docs/functionality/rosetta/hotkeys.md:41]: Can you point me to a version that supports it?

**Review by sesi200 — CHANGES_REQUESTED** [review]

**Review by MarioDfinity — APPROVED** [review]

**Review by sesi200 — APPROVED** [review]

File: docs/developer-docs/functionality/rosetta/hotkeys.md (hunk 3 above)
Comment (sesi200) [reply docs/developer-docs/functionality/rosetta/hotkeys.md:96]: It would be nice to mention this in the text, but I'm fine merging as-is too.

Comment (jean-dfinity) [reply docs/developer-docs/functionality/rosetta/hotkeys.md]: True, added the comment.

**Pull request merged**